### ISSUE TITLE
Event location issue on the calendar

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#2347]
We should either reach out the party responsible or figure out a way to fix the extra “MN” added to addresses that are not in MN. We should also look into the double addresses that appear for St. Olaf locations.

1 | 2
--|--
<img src="https://i.imgur.com/yYNh8YS.jpg" /> | <img src="https://i.imgur.com/zo0mA06.jpg" />

**Comment by hawkrives** [comment]
To follow up on this: It makes no sense.

Take this event:

<img width=320 src=https://user-images.githubusercontent.com/464441/36361500-05d1ea34-14f2-11e8-94de-274df86eba5d.PNG />

In Google, it looks like this:

<img width="320" alt="screen shot 2018-02-18 at 9 21 41 pm" src="https://user-images.githubusercontent.com/464441/36361502-0f4e596c-14f2-11e8-8285-9fe7bf93572e.png">

On the calendar, though, we get _no location_: https://wp.stolaf.edu/calendar/event/class-and-lab-open-again-for-editing-2018-19-courses/

In the calendar's iCal feed, which is what Google consumes, we _do_ have a location:

```ical
BEGIN:VEVENT
DTSTART;VALUE=DATE:20180219
DTEND;VALUE=DATE:20180220
DTSTAMP:20180219T032214
CREATED:20171211T165108Z
LAST-MODIFIED:20180218T231046Z
UID:12053-1518998400-1519084799@wp.stolaf.edu
SUMMARY:Class and Lab open again for editing 2018-19 courses
DESCRIPTION:
URL:https://wp.stolaf.edu/calendar/event/class-and-lab-open-again-for-editing-2018-19-courses/
LOCATION:MN
CATEGORIES:Academic Year
END:VEVENT
```

But in the calendar admin, there's no location!

<img width="594" alt="screen shot 2018-02-18 at 9 25 12 pm" src="https://user-images.githubusercontent.com/464441/36361518-3e8e97aa-14f2-11e8-8e97-4988778ab806.png">

The plugin that adds "year export" functionality to the calendar doesn't have it:

```php
<?php
/*
Plugin Name: The Events Calendar Year Export
Description: The Events Calendar Get Events for 1 Year from Today in iCal Export File. Trigger with link: http://example.com/events/?ical=1&year-feed
Version: 1.0
Author: Cliff from MT Support
Plugin URI: https://www.stolaf.edu/multimedia/help-wp://gist.github.com/cliffordp/ab1f7c4d95723ee6f892/
*/

// From https://gist.github.com/cliffordp/ab1f7c4d95723ee6f892/, a fork of https://gist.github.com/jesseeproductions/2b1af6527b7029eaea6e
// References:
  // https://theeventscalendar.com/support/forums/topic/ical-only-pushing-1-month-at-a-time/
  // https://theeventscalendar.com/support/forums/topic/how-users-can-subscribe-to-my-events-calendar-in-their-personal-calendars/#post-1022932
  // https://tribe.uservoice.com/forums/195723-feature-ideas/suggestions/3777092-subscribe-to-calendar-via-ical
/*
* The Events Calendar Get Events for 1 Year from Today in iCal Export File
* add coding to child theme's functions.php
* Tested works with The Events Calendar v3.12 and v4.0
* trigger export with link: http://example.com/events/?ical=1&year-feed
* change 365 for a different range
*/
// See https://gist.github.com/cliffordp/b0e5ece86040d9838f09 to customize text of iCal Event Export button text
// See https://gist.github.com/cliffordp/0add8931e1c6422a7b4c to override the iCal link to always be this link
add_action( 'pre_get_posts', 'cliff_tribe_custom_time_range_ics_export' );
function cliff_tribe_custom_time_range_ics_export( WP_Query $query ) {
        if ( ! isset( $_GET['ical'] ) || ! isset( $_GET['year-feed'] ) ) {
                return;
        }
        if ( ! isset( $query->tribe_is_event_query ) || ! $query->tribe_is_event_query ) {
                return;
        }

        $query->set( 'eventDisplay',  'custom' );
        $query->set( 'start_date',    'now' );
        $query->set( 'end_date',      '+ 365 days' ); // http://php.net/manual/en/datetime.formats.relative.php
        $query->set( 'posts_per_page', -1 );
}
```

---


TL;DR: Somewhere between the calendar and the iCal export provided by the calendar, events are gaining an "MN" location.

**Comment by drewvolz** [comment]
@hawkrives thanks for fixing.
